README.md

```diff
@@ -12,22 +12,14 @@ pip install octodns_selectel
 
 #### requirements.txt/setup.py
 
-Pinning specific versions or SHAs is recommended to avoid unplanned upgrades.
+Pinning specific versions is recommended to avoid unplanned upgrades.
 
 ##### Versions
 
 ```
 # Start with the latest versions and don't just copy what's here
-octodns==0.9.14
-octodns_selectel==0.0.1
-```
-
-##### SHAs
-
-```
-# Start with the latest/specific versions and don't just copy what's here
--e git+https://git@github.com/octodns/octodns.git@9da19749e28f68407a1c246dfdf65663cdc1c422#egg=octodns
--e git+https://git@github.com/octodns/octodns-selectel.git@ec9661f8b335241ae4746eea467a8509205e6a30#egg=octodns_selectel
+octodns==0.9.17
+octodns_selectel==0.0.3
```

Comment:
What's the reason for these two changes? In general there's a decent number of people asking when releases are going to be made/wondering how to get specific unreleased functionality. The SHA bit here and up above is there to help support that use case without them having to file an issue/me respond with how. I added them after a few rounds of that question coming up.

Reply:
I thought it seems too specific and confusing with "don't just copy what's here".  
But you definitely know best what users struggle with so i rolledback ths detaill